taxonomies/ingredients.txt

```diff
@@ -29864,7 +29903,7 @@ cy:Soia
 da:soja
 de:Soja
 el:σόγια
-es:soja, soya
+es:soja, soya, soja ecológica
```

Comment:
ecológica -> we should use the labels taxonomy for that, and not add "XYZ soja ecológica" entries.

Reply:
It doesn't get processed in the ingredients and results in es:soja-ecologica in the ingredients_tags.